ui/desktop/electron-app/src/index.js

```diff
@@ -186,6 +186,11 @@ app.on('ready', async () => {
   // per Electronegativity PERMISSION_REQUEST_HANDLER_GLOBAL_CHECK
   ses.setPermissionRequestHandler((webContents, permission, callback) => {
     /* eng-disable PERMISSION_REQUEST_HANDLER_JS_CHECK */
+    //we need to allow this for the hds clipboard action to not fail
```

Comment:
```suggestion
    // We need to allow this for native clipboard usage
```

We will likely also need this when fixing the copy paste issue for the embedded terminal in windows so just wanted to adjust the comment